### ISSUE TITLE
Add porter install command

### ISSUE THREAD

**Issue by carolynvs** [deislabs/porter#106]
This should link to functionality behind `duffle install` so that a user doesn't have to switch between porter and duffle. No code copy/pasta!

UPDATE: 

I've built out more of the commands that need to be implemented for this epic. So for this issue in particular I'm going to start with just 

```
porter install --insecure [-f path/to/bundle.json] [--name CLAIM] [-c CREDS]`
```

* `--insecure` has to be specified for now since #224 isn't implemented yet.
* `--name` will default to the bundle name, and is an optional flag.
* `-f` will default to the bundle file in the current direction, just like it does for `porter run`.
* the creds will still be handled by duffle until porter has that implemented.

The goal is that a developer could have a feedback cycle of `porter build` -> `porter install` -> edit ...

Tasks:
- [x] scaffold the command (#229)
- [x] call out to duffle for the install
- [ ] read creds from our own store in PORTER_HOME
- [x] --name defaults to the name of the bundle in the manifest
- [x] -f defaults to the bundle in the current directory

**Comment by radu-matei** [comment]
So if no code 🍝, would `duffle` be a required dependency for `porter install`?

**Comment by carolynvs** [comment]
The idea is that duffle would organize its code into packages that are reusable so that others (like porter) could include its functionality into their own clients. Duffle libraries would be a compile time dependency, not a runtime dependency. The end-user wouldn't be aware of duffle at all and wouldn't install it.

**Comment by carolynvs** [comment]
This is still in the "will it blend" stage of idea. So please feel free to slap me with fish (oops I think you missed that in standup and there's not much context in this issue! Yes, fish slapping reality checks are encouraged! 🐟 )

I am exploring different UX options for working with both duffle and porter.

1. put porter _into_ duffle. Kind of rejected when I talked to people at Kubecon because duffle is a ref. impl. and porter is opinionated.
2. put duffle _into_ porter. Reorg the packages a bit so that they can be reused by other clients. This lets someone use porter as a single client and never know what duffle is.
3. Suggest more options here

**Comment by jeremyrickard** [comment]
I think reorganizing Duffe so we could be for the useful things would be the best approach. (This would be #2 above, right?)

**Comment by carolynvs** [comment]
yeah reorganizing the code in duffle so that useful code is exported is #2 from the options above.

**Comment by vdice** [comment]
I'd say 2 feels best for short-to-mid-term for me as well.  (Spec and therefore reference implementation still young/evolving, definitely wise to avoid duplication and to do so *should* use main reference implementation, etc.)

**Comment by carolynvs** [comment]
Implemented in #233
